Library/Homebrew/service.rb

```diff
@@ -399,6 +399,7 @@ def manual_command
                                    .map { |k, v| "#{k}=\"#{v}\"" }
 
       out = vars + command if command?
+      out.map! { |arg| arg.include?(" ") ? "'#{arg}'" : arg }
```

Comment:
Maybe `sh_quote` is better?

Reply:
that outputs `/opt/homebrew/opt/nginx/bin/nginx -g daemon\ off\;` which looks less pretty to be but is correct I guess?

Reply:
Pushed in 60cdc60b7

Reply:
I don't think that one is correct, I believe it should be `/opt/homebrew/opt/nginx/bin/nginx -g 'daemon off;'`

Reply:
@SMillerDev these two are equivalent

Reply:
I believe nginx specifically complained. But if you say it works that's fine.

Reply:
@SMillerDev yeh, seems to work fine here, both work the same.